reconcile/ocm_upgrade_scheduler.py

```diff
@@ -297,25 +350,37 @@ def act(dry_run, diffs, ocm_map):
     for diff in diffs:
         action = diff.pop("action")
         cluster = diff.pop("cluster")
+        gates_to_agree = diff.pop("gates_to_agreee")
         ocm = ocm_map.get(cluster)
         if action == "create":
             logging.info([action, cluster, diff["version"], diff["next_run"]])
             if not dry_run:
+                if len(gates_to_agree) > 0:
+                    agreement = ocm.create_version_agreement(gates_to_agree)
+                    if agreement.get("version_gate") is None:
+                        # What should happen, error message?
+                        continue
                 ocm.create_upgrade_policy(cluster, diff)
         elif action == "delete":
             logging.info([action, cluster, diff["version"]])
             if not dry_run:
                 ocm.delete_upgrade_policy(cluster, diff)
 
 
-def run(dry_run, gitlab_project_id=None, thread_pool_size=10):
```

Comment:
Parameters are unused

Reply:
they are unused in the current format we are running the integration (via SQS gateway).
all such integrations should maintain an ability to run directly against gitlab, hence the need to retain the ability to supply a gitlab project id.